scene/main/viewport.h

```diff
@@ -271,7 +272,12 @@ class Viewport : public Node {
 	bool handle_input_locally = true;
 	bool local_input_handled = false;
 
+	// Collider to frame
 	Map<ObjectID, uint64_t> physics_2d_mouseover;
+	// Collider & shape to frame
+	Map<Pair<ObjectID, int>, uint64_t, PairSort<ObjectID, int>> physics_2d_shape_mouseover;
+	// Cleans up colliders corresponding to old frames or all of them.
+	void _cleanup_mouseover_colliders(bool clean_all_frames, bool p_paused_only, uint64_t frame_reference = 0);
```

Comment:
clean_all_frames and frame_reference are missing `p_` prefix, sorry I missed it before.

Reply:
Done, thanks for catching that. Am I right that the convention is to always append `p_` to method arguments (as opposed to fields/local variables)?